src/sdk/pynni/nni/compression/torch/builtin_pruners.py

```diff
@@ -5,7 +5,8 @@
 import torch
 from .compressor import Pruner
 
-__all__ = ['LevelPruner', 'AGP_Pruner', 'FPGMPruner', 'L1FilterPruner', 'SlimPruner']
+__all__ = ['LevelPruner', 'AGP_Pruner', 'SlimPruner', 'RankFilterPruner', 'L1FilterPruner', 'L2FilterPruner',
```

Comment:
do we still need `RankFilterPruner`?

Reply:
No it should be `FPGMPruner`, it's my mistake...

Reply:
not updated yet?

Reply:
updated.